### PR TITLE
fix \textquoteright + indentation

### PR DESCRIPTION
**Pull request by GinoGiotto**
These two latex substitutions:

<code>latexdef "'''" as "\textquoteright\textquoteright\textquoteright";</code>
<code>latexdef "''''" as "\textquoteright\textquoteright\textquoteright\textquoteright";</code>

Generate the error: <code>Double superscript. \end{align}</code>

So I replaced them with:

<code>latexdef "'''" as "\textquoteright{}\textquoteright{}\textquoteright{}";</code>
<code>latexdef "''''" as "\textquoteright{}\textquoteright{}\textquoteright{}\textquoteright{}";</code>

And the error no longer appears.

I also changed a few indentations because it makes my job easier at collecting latexdefs and check them. I don't think this should change anything for you, but let me know if you are ok with this.

### CODE REVIEW

**Review by tirix — APPROVED** [review]
Changes are fine with me.
I have not checked the LaTeX correction, so  I trust you with that change.

**Comment by benjub** [comment]
Why not the much simpler and with very similar display
```
latexdef "'''" as "'''";
latexdef "''''" as "''''";
```
?

File: set.mm
Comment (benjub) [inline set.mm:433655]:
```suggestion
```
no need to keep the old definition

File: set.mm
Comment (benjub) [inline set.mm:434127]:
```suggestion
```

File: set.mm
Comment (benjub) [inline set.mm:437338]:
```suggestion
 latexdef "'''" as "'''";
```

File: set.mm
Comment (benjub) [inline set.mm:437342]:
```suggestion
 "''''";
```

Comment (benjub) [reply set.mm:437342]: (note that it fits in a single line)

**Review by tirix — CHANGES_REQUESTED** [review]
I think Benoît has good points, I'll hold this approval for now! :-)

File: set.mm
Comment (GinoGiotto) [reply set.mm:433655]: I agree, if this general policy is valid for all latexdefs, I might do a pull request to remove them.

File: set.mm
Comment (GinoGiotto) [reply set.mm:437342]: Done

File: set.mm
Comment (GinoGiotto) [reply set.mm:437338]: Done

File: set.mm
Comment (GinoGiotto) [reply set.mm:434127]: Done

**Review by tirix — APPROVED** [review]

File: set.mm
Comment (benjub) [reply set.mm:433655]: Maybe not general, since sometimes it's a matter of choice. Here, the present definition is clearly better than the old one.

**Comment by GinoGiotto** [comment]
I missed a little adjustment (sorry for that), should be good now

File: set.mm
Comment (GinoGiotto) [reply set.mm:433655]: ok

**Comment by tirix** [comment]
Still looks good. Let's wait for @benjub's review and then we're good to merge.

**Comment by benjub** [comment]
As in the other PR, I cannot apply the suggested change, so it'd be nice if @GinoGiotto can.

**Comment by tirix** [comment]
I think he already applied all changes.

**Comment by GinoGiotto** [comment]
> As in the other PR, I cannot apply the suggested change, so it'd be nice if @GinoGiotto can.

@benjub I don't understand. In the commmit 0106361 and 485019c I did the requested changes

**Review by benjub — APPROVED** [review]

**Pull request merged**
